### ISSUE TITLE
Web UI crash

### ISSUE THREAD

**Issue by crishoj** [nicolargo/glances#2926]
**Describe the bug**

After running `glances -w` I see this error upon the first HTTP request:

```
Glances Web User Interface started on http://0.0.0.0:61208/
INFO:     Started server process [168904]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://0.0.0.0:61208 (Press CTRL+C to quit)
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/uvicorn/protocols/http/h11_impl.py", line 373, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "/usr/lib/python3/dist-packages/uvicorn/middleware/proxy_headers.py", line 75, in __call__
    return await self.app(scope, receive, send)
  File "/usr/lib/python3/dist-packages/fastapi/applications.py", line 199, in __call__
    await super().__call__(scope, receive, send)
  File "/usr/lib/python3/dist-packages/starlette/applications.py", line 119, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/usr/lib/python3/dist-packages/starlette/middleware/errors.py", line 181, in __call__
    raise exc
  File "/usr/lib/python3/dist-packages/starlette/middleware/errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
  File "/usr/lib/python3/dist-packages/starlette/middleware/gzip.py", line 26, in __call__
    await self.app(scope, receive, send)
  File "/usr/lib/python3/dist-packages/starlette/middleware/cors.py", line 84, in __call__
    await self.app(scope, receive, send)
  File "/usr/lib/python3/dist-packages/starlette/exceptions.py", line 87, in __call__
    raise exc
  File "/usr/lib/python3/dist-packages/starlette/exceptions.py", line 76, in __call__
    await self.app(scope, receive, sender)
  File "/usr/lib/python3/dist-packages/starlette/routing.py", line 659, in __call__
    await route.handle(scope, receive, send)
  File "/usr/lib/python3/dist-packages/starlette/routing.py", line 259, in handle
    await self.app(scope, receive, send)
  File "/usr/lib/python3/dist-packages/starlette/routing.py", line 61, in app
    response = await func(request)
  File "/usr/lib/python3/dist-packages/fastapi/routing.py", line 201, in app
    raw_response = await run_endpoint_function(
  File "/usr/lib/python3/dist-packages/fastapi/routing.py", line 150, in run_endpoint_function
    return await run_in_threadpool(dependant.call, **values)
  File "/usr/lib/python3/dist-packages/starlette/concurrency.py", line 45, in run_in_threadpool
    return await anyio.to_thread.run_sync(func, *args)
  File "/usr/lib/python3/dist-packages/anyio/to_thread.py", line 28, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(func, *args, cancellable=cancellable,
  File "/usr/lib/python3/dist-packages/anyio/_backends/_asyncio.py", line 818, in run_sync_in_worker_thread
    return await future
  File "/usr/lib/python3/dist-packages/anyio/_backends/_asyncio.py", line 754, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.10/dist-packages/glances/outputs/glances_restful_api.py", line 469, in _api_all_views
    return ORJSONResponse(limits)
  File "/usr/lib/python3/dist-packages/starlette/responses.py", line 49, in __init__
    self.body = self.render(content)
  File "/usr/lib/python3/dist-packages/fastapi/responses.py", line 33, in render
    return orjson.dumps(content)
TypeError: Dict key must be str
```

**Environement (please complete the following information)**
 - Operating System: `Ubuntu 22.04.4 LTS`
 - Glances & psutil versions: 
```
Glances version:        4.1.2
Glances API version:    4
PsUtil version:         5.9.0
```
 - How do you install Glances: `sudo pip3 install --upgrade glances`
 - Glances test: `OK`

**Additional context**

* the Glances configuration file: https://pastebin.com/8HjaVHnL

**Comment by nicolargo** [comment]
Not reproduced on my side, can you please copy/paste the result of the following command line:

    glances --issue

Thanks

**Comment by crishoj** [comment]
Thanks for looking into this. `glances --issue` output below:

[glances-issue.txt](https://github.com/user-attachments/files/16736268/glances-issue.txt)

**Comment by nicolargo** [comment]
Strange...

Can you run Glances with the following option:

    glances -w --disable-webui

Then copy/paste the result of the command line:

    curl http://0.0.0.0:61208/api/4/all/limits

If it fail copy/paste the results of all the followings commands lines:

```
curl http://0.0.0.0:61208/api/4/alert/limits
curl http://0.0.0.0:61208/api/4/ports/limits
curl http://0.0.0.0:61208/api/4/diskio/limits
curl http://0.0.0.0:61208/api/4/containers/limits
curl http://0.0.0.0:61208/api/4/processcount/limits
curl http://0.0.0.0:61208/api/4/gpu/limits
curl http://0.0.0.0:61208/api/4/percpu/limits
curl http://0.0.0.0:61208/api/4/vms/limits
curl http://0.0.0.0:61208/api/4/system/limits
curl http://0.0.0.0:61208/api/4/network/limits
curl http://0.0.0.0:61208/api/4/cpu/limits
curl http://0.0.0.0:61208/api/4/amps/limits
curl http://0.0.0.0:61208/api/4/processlist/limits
curl http://0.0.0.0:61208/api/4/load/limits
curl http://0.0.0.0:61208/api/4/sensors/limits
curl http://0.0.0.0:61208/api/4/uptime/limits
curl http://0.0.0.0:61208/api/4/now/limits
curl http://0.0.0.0:61208/api/4/fs/limits
curl http://0.0.0.0:61208/api/4/wifi/limits
curl http://0.0.0.0:61208/api/4/ip/limits
curl http://0.0.0.0:61208/api/4/help/limits
curl http://0.0.0.0:61208/api/4/version/limits
curl http://0.0.0.0:61208/api/4/psutilversion/limits
curl http://0.0.0.0:61208/api/4/core/limits
curl http://0.0.0.0:61208/api/4/mem/limits
curl http://0.0.0.0:61208/api/4/folders/limits
curl http://0.0.0.0:61208/api/4/quicklook/limits
curl http://0.0.0.0:61208/api/4/memswap/limits
```

Thanks !

**Comment by nielsproest** [comment]
I can confirm this happens on Ubuntu 22.04 if i use the system packages python3-fastapi and uvicorn instead of the latest pip versions.

**Comment by nicolargo** [comment]
@crishoj What's yoour FastAPI Lib version ?

    pip freeze

**Comment by crishoj** [comment]
```
> pip freeze | grep fastapi
fastapi==0.63.0
```

**Comment by crishoj** [comment]
I can confirm that the issue is resolved by upgrading to `fastapi` 0.112.2.

**Comment by nicolargo** [comment]
Reproduced with all version < 0.82.
